source/manual/emergency-alerts.html.md

```diff
@@ -1,25 +1,25 @@
 ---
-owner_slack: "#govuk-notify-emergency-alerts"
-title: What is GOV.UK Notify Emergency Alerts?
+owner_slack: "#emergency-alerts"
+title: What is GOV.UK Emergency Alerts?
 layout: manual_layout
 type: learn
 parent: "/manual.html"
-section: Notify Emergency Alerts
+section: Emergency Alerts
 important: true
 ---
 
-The Cabinet Office and the Department for Digital, Culture, Media & Sport are developing a ‘cell broadcast’ alert system to enable people whose lives are at risk in an emergency to be rapidly contacted via their mobile phone. They are currently at the testing phase and subject to successful progress, hope to launch the service soon.
+The Cabinet Office COBR Unit and the Department for Science, Innovation and Technology have developed an emergency alert system to enable people whose lives are at risk in an emergency to be rapidly contacted via their mobile phone. It is currently in the trial phase following a successful national test message.  it was originally developed by the GOV.UK Notify team and is now managed and maintained by the GOV.UK Emergency Alerts team.
```

Comment:
It looks wrong to me, I assumed you meant COBRA, so have linked to the explainer 👍 

```suggestion
The Cabinet Office [COBR](https://www.instituteforgovernment.org.uk/explainer/cobr-cobra) Unit and the Department for Science, Innovation and Technology have developed an emergency alert system to enable people whose lives are at risk in an emergency to be rapidly contacted via their mobile phone. It is currently in the trial phase following a successful national test message. It was originally developed by the GOV.UK Notify team and is now managed and maintained by the GOV.UK Emergency Alerts team.
```